spec/update/redownload_spec.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-RSpec.describe "bundle update" do
+RSpec.describe "bundle update", :bundler => "< 3" do
```

Comment:
The `--redownload` flag to `bundle update` should also be tested in bundler 3 mode.